lib/tasks/ci.rake

```diff
@@ -13,6 +13,30 @@ if Rails.env.development? || Rails.env.test?
     sh "rspec"
   end
 
+  desc "Run CI rspec tests"
+  task ci_rspec_tests: %i[environment rspec_tests] do
+    puts "CI rspec tests"
+    puts Rainbow("PASSED").green
+    puts ""
+  rescue StandardError => e
+    puts e.to_s
+    puts Rainbow("FAILED").red
+    puts ""
+    raise(e)
+  end
+
+  desc "Run CI js_tests"
+  task ci_js_tests: %i[environment js_tests] do
+    puts "CI js_tests"
+    puts Rainbow("PASSED").green
+    puts ""
+  rescue StandardError => e
+    puts e.to_s
+    puts Rainbow("FAILED").red
+    puts ""
+    raise(e)
+  end
+
```

Comment:
I think these tasks could go under `ci` namespace so that we get these tasks:
- `ci`: runs `ci:all`
- `ci:all` runs all other ci tasks
- `ci:js` runs `js_tests`
- `ci:rspec` runs rspec